### PR TITLE
Single settings page with improved access key management

### PR DESCRIPTION
**Pull request by jmorganca**
This PR simplifies settings and access key management to be in one place:

1. Account and org settings are now under "Settings"
2. Access keys is split into 1/ Connector and 2/ Personal access keys
3. The Dashboard's top navbar has been replaced with user info on the bottom left

Screenshot 2022-12-02 at 11 01 50 PM

Left to do:

- [x] Hide dashboard session access token.

### CODE REVIEW

File: ui/pages/settings/index.js [hunk 1]
```diff
@@ -42,109 +551,223 @@ function AdminList({ grants, users, groups, onRemove, auth, selfGroups }) {
...
+                    >
+                      Revoke
+                      <span className='sr-only'>{info.row.original.name}</span>
+                    </button>
+                    <DeleteModal
+                      open={open}
+                      setOpen={setOpen}
+                      primaryButtonText='Revoke'
+                      onSubmit={async () => {
+                        await fetch(`/api/grants/${deleteId}`, {
```
Comment (BruceMacD) [inline ui/pages/settings/index.js:625]: should we be handling errors here?

**Comment by mxyng** [comment]
Instead of splitting into Connector and Personal Keys, it might make more sense to split into Organization and Personal Keys. Organization keys include connector keys but also SCIM keys.

**Review by BruceMacD — APPROVED** [review]
this seems good to merge

**Review by pdevine — APPROVED** [review]
lgtm

**Pull request merged**